main/core/src/mill/define/ParseArgs.scala

```diff
@@ -31,14 +31,14 @@ object ParseArgs {
 
   @deprecated("Use apply(Seq[String], SelectMode) instead", "mill after 0.10.0-M3")
   def apply(
-      scriptArgs: Seq[String],
-      multiSelect: Boolean
-  ): Either[String, TargetsWithParams] = extractAndValidate(scriptArgs, multiSelect)
+             scriptArgs: Seq[String],
+             multiSelect: Boolean
+           ): Either[String, TargetsWithParams] = extractAndValidate(scriptArgs, multiSelect)
```

Comment:
Unnecessary format change. And some more in this file.

Reply:
Hah that's intellij messing things up during copy-paste, will fix